packages/react/src/index.ts

```diff
@@ -84,14 +84,55 @@ export type {
   ReadResultWithError,
   ReadResultWithoutError,
 } from './helpers/reads';
-export { LimitType } from './helpers/reads';
 export * from './helpers/tasks';
 
 /**
  * GQL common types
  */
 export type { App } from '@lens-protocol/api-bindings';
-export { TriStateValue } from '@lens-protocol/api-bindings';
+
```

Comment:
could you help to identify which of these should NOT be exported? I just copied the list from the client SDK and removed a few I know are internal for react sdk